data/syndicate jobs.txt

```diff
@@ -306,3 +306,183 @@ ship "Vanguard" "Vanguard Test Dummy"
 	add attributes
 		"automaton" 1
 		"self destruct" 1
+
+
+mission "Syndicate Prisoner Transport [0]"
+	name "Transport indicted executive to house arrest"
+	job
+	repeat
+	description `Transport an indicted white-collar criminal with his assistants, lawyers, and collection of <cargo> to <planet>, where he will await trial under "house arrest." Payment is <payment>.`
+	cargo "Luxury Goods" 1 10
+	passengers 5 12
+	source
+		government "Syndicate"
+		attributes "urban"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "rich"
+		attributes "frontier"
+	to offer
+		random < 5
+	on offer
+		require "Luxury Accommodations"
+	on complete
+		payment 10000 200
+		dialog "The executive strolls off your ship with a sunny smile on his face. His assistants arrange to have his <cargo> removed from your ship. One of the lawyers hands you your payment of <payment> with muttered threats about libel lawsuits if you say anything about what you may or may not have overheard on your ship."
+
+
+mission "Syndicate Prisoner Transport [1]"
+	name "Transport convicted saboteur"
+	job
+	repeat
+	description `Transport a convicted industrial saboteur to <planet> for their sentence. Payment is <payment>.`
+	passengers 1
+	source
+		government "Syndicate"
+		not attributes "military"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "military"
+	to offer
+		random < 15
+	on offer
+		require "Brig"
+	on complete
+		payment 10000 200
+		dialog "A group of guards carry the prisoner off your ship more roughly than is strictly necessary. You receive your payment of <payment>."
+
+
+mission "Syndicate Prisoner Transport [2]"
+	name "Transport convicted corporate spies"
+	job
+	repeat
+	description `Transport a group of corporate spies to <planet> for their sentences. Payment is <payment>.`
```

Comment:
Perhaps `Transport a group of [convicted] corporate spies` just to reinforce the fact that these spies have been caught? I typically skim the description of jobs instead of reading the job name, so thorough descriptions of what exactly is going on are helpful.

Reply:
Good suggestion, done in ac45a37.